src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+public class Task {
+	private boolean isDone;
+	private String taskContent;
+
+	public Task(String taskContent) {
+		this.taskContent = taskContent;
+		this.isDone = false;
+	}
+	public void markAsDone() {
+		this.isDone = true;
+	}
+	public void test() throws DukeException {
+		if (taskContent.length() == 0) {
+			throw new DukeException("☹ OOPS!!! The description of a task cannot be empty.");
+		}
+	}
+	public String getType() {
+		return "";
+	}
+	public boolean isDone() {
+		return this.isDone;
+	}
+	public String  getContent() {
```

Comment:
I think there should be only one white space between "String" and "getContent()". Perhaps using the checkstyle file will help detect such errors?